app/Util/storage/safeDeleteDirectory.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace App\Util\Storage;
```

Comment:
The parent directory should be `Storage` (like app/Util/Storage), not `storage` as that isn't PSR-4 compatible. I don't see any usage of this in this PR, can we safely close it now you made #6377